logger/detailed_logger.js

```
@@ -43,8 +43,9 @@ class DetailedLogger {
     );
   }
 
-  error(message) {
-    console.error(chalk.red(`❌ ${message}`));
+  error(message, error) {
```

error parameter is being passed in calls, but was not used here, so the reason for the error was lost.